Monika After Story/game/script-songs.rpy

```diff
@@ -595,3 +595,31 @@ label mas_song_stand_by_me_long:
     m 4euo "{i}~Whenever you're in trouble won't you stand by me~{/i}"
     m 4hub "{i}~Oh stand by me, won't you stand now, stand by me~{/i}"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_songs_database,
+            eventlabel="mas_song_rewrite_the_stars",
+            category=[store.mas_songs.TYPE_SHORT],
+            prompt="Rewrite The Stars",
+            random=True,
+            aff_range=(mas_aff.NORMAL,None)
+        ),
+        code="SNG"
+    )
+
+label mas_song_rewrite_the_stars:
+    m 3ekbfb "{i}~What if we rewrite the stars~{/i}"
+    m 3ekbfa "{i}~Say you were made to be mine~{/i}"
+    m 2fkbsa "{i}~Nothing could keep us apart~{/i}"
+    m 3hubfb "{i}~Cause you are the one i was meant to find~{/i}"
```

Comment:
```suggestion
    m 3hubfb "{i}~You'd be the one i was meant to find~{/i}"
```

Reply:
You don't need to review. Just push to your branch
![image](https://user-images.githubusercontent.com/22531674/74391913-869b8a00-4dd3-11ea-95e3-a7e1d1334207.png)
![image](https://user-images.githubusercontent.com/22531674/74391940-9915c380-4dd3-11ea-8531-983ea76c8a3b.png)

Reply:
Ok, sorry, still kinda dont understand how github works